### ISSUE TITLE
ENOENT after 0.5.0 upgrade [solved?]

### ISSUE THREAD

**Issue by rhettl** [kaermorchen/ember-cli-bootstrap-4#26]
Hello, 

I fixed this issue for me, so this is just for others who may have issues. 

I should have recorded the full stack, but after upgrading to `0.5.0` (needed to fix `.table-responsive`) I couldn't build anymore. I received the below error message.

```
Error: ENOENT: no such file or directory, open '/Users/rhett/Projects/mat-interviews-frontend/tmp/source_map_concat-input_base_path-ITxFXnZx.tmp/node_modules/popper.js/dist/umd/popper.js'
```

Project seems to be working after
1. update to ember-cli@2.18.0  -- `npm i -g ember-cli@2.18.0`
2. reinstall bower_components, tmp, dist, and npm_modules -- `rm -rf bower_components tmp dist node_modules && bower install`
3. reinitialize ember with new cli -- `ember init` then use diff-ing and editing to merge my core files with new files.
4. all is good

I hope someone finds a less complicated solution, but this worked for me. 

THANK YOU for all the great work you do!

**Comment by rhettl** [comment]
PS: before this final solution, I did try `rm -rf dist tmp bower_components node_modules && bower install && npm i && npm start` but no change

**Comment by potato20** [comment]
Getting the same issue  here

**Comment by potato20** [comment]
i fixed it simply by upgrading to ember-cli 2.18. didn't do or change anything else.

**Comment by kaermorchen** [comment]
Yes I think this is solved.
